corebehrt/modules/setup/directory.py

```
@@ -76,6 +76,8 @@ def get_config_path(self, directory: str, name: str = None) -> str:
                 "tokenized": DATA_CFG,
                 "prepare_pretrain": PREPARE_PRETRAIN_CFG,
                 "prepare_finetune": PREPARE_FINETUNE_CFG,
+                "prepare_pretrain": PREPARE_PRETRAIN_CFG,
```

Ruff (0.8.2)

Dictionary key literal `"prepare_pretrain"` repeated

Remove repeated key literal `"prepare_pretrain"`

(F601)